webapp/src/dogma/common/components/Filter.tsx

```diff
@@ -0,0 +1,27 @@
+import { Column } from '@tanstack/react-table';
+import { DebouncedInput } from '@/dogma/common/components/DebouncedInput';
+import { useMemo } from 'react';
+
+export const Filter = ({ column }: { column: Column<any, unknown> }) => {
```

Comment:
What do you think of creating a new folder such as `table` and moving the new components to it?